src/Sulu/Bundle/PreviewBundle/Preview/Exception/RouteDefaultsProviderNotFoundException.php

```diff
@@ -20,7 +20,7 @@ class RouteDefaultsProviderNotFoundException extends PreviewRendererException
      * @param int|string $id
      * @param string $locale
      */
-    public function __construct($object, $id, $webspaceKey, $locale)
+    public function __construct(object $object, $id, $webspaceKey, $locale)
```

Comment:
I think this is mixed in some cases we return `array` only as a workaround if I remember correctly 🤔

Reply:
okay reverted as the caller already checks for the object class